tests/framework-tests/preferences/js/EnactorsTests.js

```diff
@@ -17,18 +17,27 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
 
     fluid.registerNamespace("fluid.tests");
 
-    fluid.tests.testStyle = function (that, container, expectedDefaultFlag, expectedCssClass) {
-        var elements = that.getElements();
+    fluid.tests.containSubstring = function (string, subString) {
```

Comment:
Should be containsSubstring - but a clearer model should use this jQuery utility http://api.jquery.com/hasclass/ rather than string-bashing